src-tauri/src/northstar/profile.rs

```diff
@@ -0,0 +1,115 @@
+use crate::util::copy_dir_all;
+use crate::GameInstall;
+
+// These folders are part of Titanfall 2 and
+// should NEVER be used as a Profile
+const SKIP_PATHS: [&str; 7] = [
+    "__overlay",
+    "bin",
+    "Core",
+    "r2",
+    "vpk",
+    "platform",
+    "Support",
+];
+
+// A profile may have one of these to be detected
+const MAY_CONTAIN: [&str; 9] = [
+    "mods/",
+    "plugins/",
+    "packages/",
+    "logs/",
+    "runtime/",
+    "save_data/",
+    "Northstar.dll",
+    "enabledmods.json",
+    "placeholder.playerdata.pdata",
```

Comment:
If you wanna get ahead of the curve you can also add `LEGAL.txt` to the list ^^

https://github.com/R2Northstar/Northstar/issues/535